concourse-ci/metadata.json

```diff
@@ -3,7 +3,7 @@
   "itemDisplayName": "Concourse CI",
   "description": "Concourse is a CI system composed of simple tools and ideas. It can express entire pipelines, integrating with arbitrary resources, or it can be used to execute one-off tasks, either locally or in another CI system. This template can help to prepare neccessary Azure resources to setup such a CI system, and make the setup more simple.",
   "summary": "This template helps you setup Bosh and Concourse CI.",
-  "githubUsername": "bingosummer",
-  "dateUpdated": "2016-08-11"
+  "githubUsername": "norshtein",
```

Comment:
are you taking ownership of this sample?

Reply:
yes. @norshtein will take the ownership of this template.